ui/hocs/withResolvedClaimRender/view.jsx

```diff
@@ -121,6 +127,28 @@ const withResolvedClaimRender = (ClaimRenderComponent: FunctionalComponentParam)
       );
     }
 
+    if (isVisibilityRestricted !== false) {
+      if (isVisibilityRestricted === undefined) {
+        return <LoadingSpinner text={__('Resolving...')} />;
+      }
+
+      return (
+        <Wrapper>
+          <div className="main--empty">
+            <Yrbl
+              title={__(isChannel ? 'Channel unavailable' : 'Content unavailable')}
```

Comment:
I think I'm a fan of Title Case - Content Unavailable

Reply:
It's a messy mix at the moment. We should decide and stick to one.  I recall there was debate, along with things like punctuation.  I think which one looks good depends on the where it is used? i.e. need separate strings instead of sharing

For this case, I'm re-using the string, so let's fix that together when we have a decision.

<img width="509" alt="image" src="https://user-images.githubusercontent.com/64950861/233783182-ab7c2b86-a584-4939-845b-a7c02ca31a56.png">